src/sampling.jl

```diff
@@ -316,12 +338,36 @@ function sample!(a::AbstractArray, x::AbstractArray; replace::Bool=true, ordered
     return x
 end
 
+
+"""
+    sample(a[, wv::WeightVec])
+
+Select a single random element of `a`. Weighted sampling is performed if
+a weighting vector `wv` is given.
+
+    sample(a, [wv::WeightVec,] n::Integer; replace=true, ordered=false)
+
+Select a random, optionally weighted sample of size `n` from an array `a`
+using a polyalgorithm. `replace` dictates whether sampling is performed
+with replacement and `order` dictates whether an ordered sample, also called
+a sequential sample, should be taken.
+
+    sample(a, [wv::WeightVec,] dims; replace=true, ordered=false)
```

Comment:
That's a good point. (I actually just learned the other day what the `Dims` type is.)